### ISSUE TITLE
Running tests of local clone of Team Org Repo, errors?

### ISSUE THREAD

**Issue by gweeyc** [nus-tic4002-AY2122s2/forum#15]
I have successfully cloned the Team Repo and did a test run of the test folder and got this Msg:-

```
> Task :test
Feb 10, 2022 9:47:38 PM seedu.address.AppParameters parse
WARNING: Invalid config path a . Using default config path.
Feb 10, 2022 9:47:40 PM seedu.address.commons.util.JsonUtil readJsonFile
INFO: Json file src\test\data\ConfigUtilTest\NonExistentFile.json not found
Feb 10, 2022 9:47:41 PM seedu.address.commons.util.JsonUtil readJsonFile
WARNING: Error reading from jsonFile file src\test\data\ConfigUtilTest\NotJsonFormatConfig.json: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'this': was expecting 'null', 'true', 'false' or NaN
 at [Source: this file is not in json format!
; line: 1, column: 5]
Feb 10, 2022 9:47:41 PM seedu.address.logic.LogicManager execute
INFO: ----------------[USER COMMAND][uicfhmowqewca]
Feb 10, 2022 9:47:41 PM seedu.address.logic.LogicManager execute
INFO: ----------------[USER COMMAND][list]
Feb 10, 2022 9:47:41 PM seedu.address.logic.LogicManager execute
INFO: ----------------[USER COMMAND][add n/Amy Bee p/11111111 e/amy@example.com a/Block 312, Amy Street 1]
Feb 10, 2022 9:47:41 PM seedu.address.logic.LogicManager execute
INFO: ----------------[USER COMMAND][delete 9]
Feb 10, 2022 9:47:42 PM seedu.address.commons.util.JsonUtil readJsonFile
INFO: Json file src\test\data\JsonAddressBookStorageTest\NonExistentFile.json not found
Feb 10, 2022 9:47:42 PM seedu.address.storage.JsonAddressBookStorage readAddressBook
INFO: Illegal values found in src\test\data\JsonAddressBookStorageTest\invalidAndValidPersonAddressBook.json: Phone numbers should only contain numbers, and it should be at least 3 digits long
Feb 10, 2022 9:47:42 PM seedu.address.commons.util.JsonUtil readJsonFile
WARNING: Error reading from jsonFile file src\test\data\JsonAddressBookStorageTest\notJsonFormatAddressBook.json: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'not': was expecting 'null', 'true', 'false' or NaN
 at [Source: not json format!
; line: 1, column: 4]
Feb 10, 2022 9:47:42 PM seedu.address.storage.JsonAddressBookStorage readAddressBook
INFO: Illegal values found in src\test\data\JsonAddressBookStorageTest\invalidPersonAddressBook.json: Names should only contain alphanumeric characters and spaces, and it should not be blank
Feb 10, 2022 9:47:42 PM seedu.address.commons.util.JsonUtil readJsonFile
WARNING: Error reading from jsonFile file src\test\data\JsonUserPrefsStorageTest\NotJsonFormatUserPrefs.json: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'Not': was expecting 'null', 'true', 'false' or NaN
 at [Source: Not a json file!
; line: 1, column: 4]
Feb 10, 2022 9:47:42 PM seedu.address.commons.util.JsonUtil readJsonFile
INFO: Json file src\test\data\JsonUserPrefsStorageTest\NonExistentFile.json not found
> Task :jacocoTestReport
BUILD SUCCESSFUL in 22s
6 actionable tasks: 4 executed, 2 up-to-date
9:47:51 PM: Execution finished ':test'.
```

What is the JsonFile? Seems not there at all? Thank you!

**Comment by vimuthm** [comment]
Hi Mr. Alvin @gweeyc ,

`.json` is a file format that follows the JavaScript Object Notation format, [here's](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON) a quick summary.

tP uses this to as the main file format for storage instead of a raw `.txt` file. The test cases are checking edge cases such as invalid json files, missing files etc. as you can see from the naming of the json files in the log you provided.

> src\test\data\ConfigUtilTest\NotJsonFormatConfig.json
> src\test\data\JsonAddressBookStorageTest\NonExistentFile.json

So, yes these test cases are _expected_ to fail. What you see is just `INFO` and `WARNING` outputs that are due to the `Logger` that is part of the tP codebase. The `Logger` essentially logs important events that might occur in the runtime of the application so that if needed, the developers can understand better what sequence of actions led to an unexpected crash for example.

Tldr; This is expected output as these are negative test cases.

**Comment by gweeyc** [comment]
Understood now, thank you for all the kind help. Appreciate. Resolved.
